.github/actions/nix/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Setup Nix
+description: Installs and setups Nix components
+
+inputs:
+  github-token:  
+    description: 'Name of the Cachix cache to use'
```

Comment:
This is incorrect.

Reply:
chore: correct description #47dd53e3c36d0940f269297a5a860534bf988991